src/cow-react/modules/limitOrders/pure/UnlockLimitOrders/styled.ts

```diff
@@ -66,8 +66,17 @@ export const List = styled.ul`
     fill: ${({ theme }) => theme.success};
   }
 
-  > li[data-icon='progress'] > span > svg > path {
-    fill: ${({ theme }) => theme.warning};
+  > li[data-is-new='true'] > span {
+    display: flex;
+    width: max-content;
```

Comment:
This is breaking the grid.
But without it, it looks worse

https://user-images.githubusercontent.com/43217/228002483-3fbc4951-a694-477d-ae40-0d48e35c8082.mov